_posts/2024-01-19-trino-2023-wrapped.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title:  "Trino 2023 wrapped"
+author: Cole Bowden
+image: /assets/blog/2023-review/wrapped.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+If [“Wrapped” is good enough for Spotify](https://www.newsroom.spotify.com/2023-wrapped/), 
+it’s good enough for Trino, right? As we look forward to a bright 2024, we can
+also take a moment to get sentimental, look back at everything we’ve
+accomplished, and reflect on the progress we’ve made. Commander Bun Bun has been
+hard at work, so if you haven’t been paying close attention to Trino or want an
```

Comment:
```suggestion
hard at work, so if you haven’t been paying close attention to Trino, or want an
```

Reply:
It's not an independent clause after so this isn't correct